packages/@coorpacademy-player-web/src/index.js

```diff
@@ -4,8 +4,8 @@ import pipe from 'lodash/fp/pipe';
 import Provider from '@coorpacademy/components/es/atom/provider';
 import {selectProgression} from '@coorpacademy/player-store/es/actions/ui/progressions';
 import {updateCoaches} from '@coorpacademy/player-store/es/actions/ui/coaches';
-import start from '@coorpacademy/player-store/es/start';
-import createStore from '@coorpacademy/player-store/es';
+import start from './start';
```

Comment:
Can you change that `import start from './start';` to something that reflects what want to `start`